src/data/set/basic.lean

```diff
@@ -711,6 +711,9 @@ lemma eq_singleton_iff_nonempty_unique_mem {s : set α} {a : α} :
   s = {a} ↔ s.nonempty ∧ ∀ x ∈ s, x = a :=
 eq_singleton_iff_unique_mem.trans $ and_congr_left $ λ H, ⟨λ h', ⟨_, h'⟩, λ ⟨x, h⟩, H x h ▸ h⟩
 
+@[simp] lemma default_coe_sort_singleton {α : Sort*} (x : α) :
```

Comment:
```suggestion
@[simp] lemma default_coe_singleton (x : α) :
```
Nothing else uses the name `coe_sort` in this file. Also, lean is translating `{α : Sort*}` to `{α : Type*}` for you, at which point you may as well just use the variable.